packages/compiler-cli/test/compliance/test_cases/r3_compiler_compliance/components_and_directives/queries/static_content_query.js

```diff
@@ -3,8 +3,8 @@ ContentQueryComponent.ɵcmp = $r3$.ɵɵdefineComponent({
   selectors: [["content-query-component"]],
   contentQueries: function ContentQueryComponent_ContentQueries(rf, ctx, dirIndex) {
     if (rf & 1) {
-    $r3$.ɵɵstaticContentQuery(dirIndex, SomeDirective, true);
-    $r3$.ɵɵcontentQuery(dirIndex, $ref0$, true);
+      $r3$.ɵɵcontentQuery(dirIndex, SomeDirective, 3);
```

Comment:
The `3` here can be expressed as a union if we declare a union flag matcher for it here:

https://github.com/angular/angular/blob/76ae87406f487adc2f3a501ca10672a7d6cde49e/packages/compiler-cli/test/compliance/test_helpers/expected_file_macros.ts#L36-L37

Doing so would allow this to read

```suggestion
      $r3$.ɵɵcontentQuery(dirIndex, SomeDirective, __QueryFlags.descendants__|__QueryFlags.isStatic__);
```

This applies to all compliance tests under `packages/compiler-cli/test/compliance` (but not compliance_old nor ngtsc/ngcc tests)

Reply:
fixed